packages/serverless/src/apollo/app/document.ts

```diff
@@ -10,48 +13,42 @@ import {
   documentRecord as schemaDocumentRecord,
   pagination,
   TDocumentCreateRequest,
-  TDocumentListRequest,
   TDocumentFindRequest,
   TDocumentUpdateRequest,
   TDocumentHistoryFindRequest,
-  TDocumentHistoryListRequest,
-  TWithProofStatus,
-  TDocumentWithMetadataResponse,
-  TDocumentRecordNullable,
-  TDocumentHistoryListResponse,
   TDocumentModificationResponse,
   TDocumentHistoryResponse,
+  TPaginationReturn,
+  TDocumentFindResponse,
+  PERMISSION_DEFAULT_VALUE,
 } from '@zkdb/common';
 
-import { DEFAULT_PAGINATION } from '@common';
 import { Permission } from '@zkdb/permission';
+import { GraphqlHelper } from 'src/helper/graphql';
```

Comment:
danger import when build js dont know where 'src/' to resolve

Reply:
fixed